app/classic/edit/index.html

```diff
@@ -1,25 +1,43 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>JupyterLite Classic - Edit</title>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="/classic/edit/favicon.ico">
-</head>
-<body>
-    <script id="jupyter-config-data" type="application/json">
-      {
-        "classicPage": "edit",
-        "baseUrl": "/",
-        "fullStaticUrl": "/classic/build",
-        "fullMathjaxUrl": "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js",
-        "mathjaxConfig": "TeX-AMS_CHTML-full,Safe",
-        "settingsUrl": "/classic/build/schemas",
-        "themesUrl": "/classic/build/themes"
-      }
+  <head>
+    <title>JupyterLite Classic - Edit</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script id="jupyter-config-data" type="application/json"></script>
+  </head>
+  <body>
+    <script>
+      (function() {
+        var pageInfo = document.getElementById('jupyter-config-data');
+        var base = window.location.pathname.replace(
+          /\/classic\/edit(\/|\/index.html)?$/,
+          '/'
+        );
+        pageInfo.textContent = JSON.stringify(
+          {
+            classicPage: 'edit',
+            baseUrl: base,
+            fullStaticUrl: base + 'classic/build',
+            fullMathjaxUrl:
+              'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js',
+            mathjaxConfig: 'TeX-AMS_CHTML-full,Safe',
+            settingsUrl: base + 'classic/build/schemas',
+            themesUrl: base + 'classic/build/themes'
+          },
+          null,
+          2
+        );
+        const bundle = document.createElement('script');
+        bundle.src = base + 'classic/build/bundle.js';
+        bundle.main = 'index';
+        document.body.appendChild(bundle);
+        const favicon = document.createElement('link');
+        favicon.rel = 'icon';
+        favicon.type = 'image/x-icon';
+        favicon.href = base + 'lab/favicon.ico';
```

Comment:
It would be nice to keep the classic favicon here (for files and notebooks):

![image](https://user-images.githubusercontent.com/591645/115108681-11e15400-9f72-11eb-9b0e-3d9a41ab72b3.png)

Reply:
will revert, but maybe we need another treatment... i think it's important the app indicates that _this ain't your regular jupyter_... maybe a book/document with a lightswitch on it?

Reply:
Sounds good :+1: